Dockerfile

```
@@ -2,6 +2,7 @@
 
 # Build the application
 FROM golang:1.20 AS build-stage
+ARG TARGETPLATFORM
```

```suggestion
ARG TARGETARCH
```
Here and below; TARGETPLATFORM will be a full `linux/arm64`, and for GOARCH we'll need `arm64` only.